content/blog/2024/transformer_progress.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "Transformer Progress"
+author: "Daniel Monroe"
+published: 2024-02-16
+---
+
+
+The transformer architecture has dominated domains as diverse as vision and language perception recently. Over the past two years, the Lc0 team has been trying to answer the following question:
+
+   *What chess-specific enhancements can be made to the transformer architecture?*
+<!--more-->
+
+Leela’s nets have historically struggled with long-range dependencies, failing to recognizing positional and tactical ideas involving squares that are far away from each other, like multiloaded pieces. This was because Leela’s networks used a convolution-based architecture, like those used in the DeepMind AlphaZero project on which Lc0 is based. Effectively, each square iteratively analyzes the information at each adjacent square and uses that information to refine its representation. The main drawback of this approach is the small receptive fields of the convolution filters. For the a1 square to “learn” about what piece is on at h8, the information must make at least 7 trips from square to square.
+
+#### Our work
+
+All of our experiments were encoder-only with a Post-LN architecture. We use 64 tokens, one for each square. The order of the tokens is flipped with the side to move. Most of our initial experiments used 10 layers with embedding size 256 and 8 heads, and a 4&times; dff expansion ratio, with mish nonlinearities in the ffn layers. The embeddings were formed as linear projections from an input consisting of:
```

Comment:
I'd suggest `Post-LN` be expanded to `Post-LayerNorm`. Also, abbreviations like DFF and FFN should be capitalized.